myserver.cpp

```diff
@@ -0,0 +1,684 @@
+#include "myserver.h"
+#include "IDataBase.cpp"
+#include "IDataBase.h"
+#include "DBErrors.h"
+#include <iostream>
+#include <vector>
+#include <string>
+
+myserver::myserver(){}
+
+myserver::~myserver(){}
+
+void myserver::startServer(){
+    if(this->listen(QHostAddress::Any,5555)){
+        qDebug()<<"Listening";
```

Comment:
Add more meaningful log messages
Not just "yes" or "no"

Reply:
Made a more informative conclusion